### PR TITLE
PDA-42 process definition list endpoint

### PR DESCRIPTION
**Pull request by ffleandro**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: src/main/java/org/entando/plugins/pda/controller/AuthPermissions.java [hunk 1]
```diff
@@ -15,4 +15,7 @@
     public static final String CONNECTION_CREATE = "connection-create";
     public static final String CONNECTION_EDIT = "connection-edit";
     public static final String CONNECTION_DELETE = "connection-delete";
+
+    public static final String PROCESS_LIST = "process-list";
+    public static final String PROCESS_GET = "process-get";
```
Comment (werbth) [inline src/main/java/org/entando/plugins/pda/controller/AuthPermissions.java]: This constant is not being used.

**Review by werbth — CHANGES_REQUESTED** [review]

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by werbth** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by ffleandro** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by ffleandro** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by werbth** [comment]
retest this please

**Comment by entando-service** [comment]
All is well

**Review by werbth — APPROVED** [review]

**Comment by werbth** [comment]
retest this please

**Comment by entando-service** [comment]
All is well

**Pull request merged**